rollup.config.js

```diff
@@ -0,0 +1,48 @@
+import uglify from 'rollup-plugin-uglify'
```

Comment:
`import { uglify } from 'rollup-plugin-uglify'`

Reply:
This branch uses [`rollup-plugin-uglify@2.0.1`](https://github.com/thysultan/stylis.js/pull/87/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R40), so it's good. If we get to move this forward then I'm gonna rebase & update deps.